### ISSUE TITLE
Text not sanitized, crashes page

### ISSUE THREAD

**Issue by ed2050** [zauberzeug/nicegui#2741]
### Description

Text input to ui.label is not properly sanitized.

Add this code anywhere on any nicegui page:
```py
ui.label ('${ abc')
```

Result: page no longer displays, entire page is blank.  JS console reports this error : 
```Uncaught SyntaxError: Missing } in template expression```

Reason: it seems nicegui loads all page text via js strings for some reason.  Generated page has a script with giant json data structure containing all page text, crammed into a js template like this: 
```js
const raw_elements = String.raw`{"0":{ ... }, "1":{ ... }, ...}`;
```

When I say giant, I mean a modest page with about 1000 chars has a `const raw_elements` line that is 54k by itself.  Seems like both the size and structure could be improved.  Why embed a string containing json on the page instead of just raw json?

In any event, input needs to be sanitized.  `${` has special meaning in js template strings.

Using nicegui version 1.3.16.  Haven't tested on newer versions, perhaps this bug was already fixed.

**Comment by falkoschindler** [comment]
Hi @ed2050,

This issue has been fixed in version 1.4.4: https://github.com/zauberzeug/nicegui/issues/2079.

The upcoming version 1.4.19 will include PR https://github.com/zauberzeug/nicegui/pull/2702 to reduce the payload.

Why do we need the string embedding? To be honest, I don't know. It might be for a reason which is long obsolete. I'll check if we can use the raw JSON instead.

**Comment by falkoschindler** [comment]
The problem with sending the raw JSON was an example like this:

```py
ui.label('</script>')
```

The closing script tag takes precedence over everything else, which breaks the JavaScript code. And instead of recursively sanitizing the JSON, we chose to serialize it and sanitize the resulting string.

See https://github.com/zauberzeug/nicegui/issues/1067 for the corresponding issue.
